styles/main.css

```diff
@@ -0,0 +1,117 @@
+@import "./theme.css";
+
+/* css reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  background-color: var(--gray1);
+  font-family: "Inter", sans-serif;
+  line-height: 1;
+  font-weight: 400;
+}
+
+.navbar {
+  height: 9.6rem;
+  display: flex;
+  padding: 0 4.8rem;
+  align-items: center;
+  color: #14213d;
```

Comment:
use variables for these color rather than using directly !!

Reply:
sure
I have updated this in the latest commit 
https://github.com/shaksham08/Sugar-UI/pull/1/commits/204d601cc4c5b329d3999d44085b8c8370cee154